DuckDuckGo/AutofillLoginDetailsView.swift

```diff
@@ -236,43 +236,6 @@ struct AutofillLoginDetailsView: View {
     }
 }
 
-struct ClearTextField: View {
```

Comment:
This makes more sense in the shared DuckUI library

Reply:
This confused me a bit because it was ultimately moved to a different file right next to this one.

That's fine, but just adding a comment to make sure that was the intention.

Reply:
My comment was incorrect, it was actually moved into its own file in the shared UI components folder, not library. DuckUI doesn’t seem to have much in it at all. Nonetheless, I thought it didn’t make much sense in the AutofillLoginDetailsView file when it’s now being used from a couple of places.